goldenmaster/Plugins/TbSimple/Source/TbSimple/Private/Generated/OLink/TbSimpleSimpleInterfaceOLinkClient.cpp

```diff
@@ -372,11 +620,31 @@ void UTbSimpleSimpleInterfaceOLinkClient::emitSignal(const std::string& signalNa
 		Execute_BroadcastSigInt(this, args[0].get<int32>());
```

Comment:
What is args here? args[0] could probably be defined somewhere and use throughout code for clarity. Counting 14 usages.

Reply:
args is a json array which on the first entry contains the signal data to be sent.
We could store it a temporary variable before broadcasting it. Would this be better?

Reply:
I think that would be good, yes.

Reply:
Create issue for this https://github.com/apigear-io/template-unreal/issues/36